addon/classes/Row.js

```diff
@@ -33,6 +35,17 @@ export default class Row extends Ember.ObjectProxy.extend({
    * @type {String | Array}
    */
   classNames: null,
+
+  /**
+   * Element ID for the corresponding `{{lt-row}}`.
+   *
+   * @property elementId
+   * @type {String}
+   * @readOnly
+   */
+  elementId: computed(function() {
```

Comment:
Note: The `Column` class will also have the same implementation once I finish the drag n drop column headers feature.